<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigN7714AWindow</class>
 <widget class="QMainWindow" name="ConfigN7714AWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Power Unit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>265</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser State</string>
    </property>
   </widget>
   <widget class="QComboBox" name="powerUnitComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>dBm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Watt</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="laserStateDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="togglePowerButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Power</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>175</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Output Power:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserOutputPowerDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="powerDisplayUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserOutputPowerEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="powerUnitEditLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Power Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wavelength Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Wavelength</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserWavelengthDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Wavelength Unit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="wavelengthUnitComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>m</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>um</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pm</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="minWavelengthDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Wavelength</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxWavelengthDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Wavelength</string>
    </property>
   </widget>
   <widget class="QLabel" name="wavelengthEditUnitLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>410</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserWavelengthEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frequency Settings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserFrequencyDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>600</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Frequency</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserFrequencyEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>600</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="frequencyEditUnitLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>610</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Frequency Unit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="frequencyUnitComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>570</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THz</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="wavelengthDisplayUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="minWavlengthUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxWavelengthUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="laserFrequencyDisplayUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>610</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="minFrequencyUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>640</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minFrequencyDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>630</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="maxFrequencyUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>670</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Frequency</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxFrequencyDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>660</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserMaxPowerDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>235</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Laser Power:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="laserMinPowerDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>205</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Laser Power:</string>
    </property>
   </widget>
   <widget class="QLabel" name="minPowerDisplayUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxPowerDisplayUnitLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(unit)</string>
    </property>
   </widget>
   <widget class="QLabel" name="instrumentInfoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>991</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Instrument: </string>
    </property>
   </widget>
   <widget class="QLabel" name="instrumentAddressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>981</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Address: </string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="slot1RadioBtn">
     <property name="text">
      <string>Slot 1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">slotRadioButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="slot2RadioBtn">
     <property name="text">
      <string>Slot 2</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">slotRadioButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="slot3RadioBtn">
     <property name="text">
      <string>Slot 3</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">slotRadioButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="slot4RadioBtn">
     <property name="text">
      <string>Slot 4</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">slotRadioButtonGroup</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="slotRadioButtonGroup"/>
 </buttongroups>
</ui>
